src/components/SingleArticle/SingleArticle.test.tsx

```diff
@@ -163,6 +165,8 @@ describe('SingleArticle component', () => {
   test('renders the article', () => {
     render(<SingleArticle article={testArticle} />)
 
+    const banner = screen.queryByText('Unpublished Article Preview')
+    expect(banner).toBeNull()
```

Comment:
Let me know if you feel this should be a separate test, but seemed to make sense here since it was testing that a published article shows.